packages/fluentui/docs/src/examples/components/Dialog/Variations/DialogExampleZoomCustomFooter.shorthand.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { Button, Dialog, Flex, FlexItem } from '@fluentui/react-northstar';
+import { useBooleanKnob } from '@fluentui/docs-components';
+import { CloseIcon } from '@fluentui/react-icons-northstar';
+
+const DialogExampleZoomContent: React.FC = () => {
+  const [open, setOpen] = useBooleanKnob({ name: 'open' });
```

Comment:
why does this example need to be controlled ?

Reply:
It doesn't need to be controlled (just copied it from another example).
Removed that. Thanks :)